src/bit-systems/object-menu.ts

```diff
@@ -81,6 +84,10 @@ function moveToTarget(world: HubsWorld, menu: EntityID) {
 //       temporary implementation that rely on the old systems.
 //       They should be rewritten more elegantly with bitecs.
 function startRotation(world: HubsWorld, targetEid: EntityID) {
+  if (hasComponent(world, Networked, targetEid)) {
+    takeOwnership(world, targetEid);
+    addComponent(world, EntityStateDirty, targetEid);
```

Comment:
It's good to `takeOwnership` right away when you `startRotation`, but you should probably wait to mark the entity state dirty until after you `endRotation`.

Reply:
Thanks. Regarding entity state dirty, should it be set only for `Networked` entity (that the client has ownership)?

Reply:
Yes -- if the entity isn't `Networked` then there's no associated entity state that can be stored on the backend.

Reply:
Updated, thanks.